packages/shell-chrome/src/backend.js

```diff
@@ -127,10 +128,9 @@ function discoverComponents(isThroughMutation = false) {
         }
 
         const data = Object.entries(rootEl.__x.getUnobservedData()).reduce((acc, [key, value]) => {
-            const type = typeof value
             acc[key] = {
-                value: type === 'function' ? 'function' : value,
-                type,
+                value: value instanceof HTMLElement ? serializeHTMLElement(value) : value,
+                type: value instanceof HTMLElement ? 'HTMLElement' : typeof value,
```

Comment:
This is a bit odd since HTMLElement isn't a real JS type

Maybe we should add a comment to look at this line wherever we're processing the message being built here